tests/schemas/api_v3/transaction.schema.js

```diff
@@ -27,6 +27,12 @@ const block = {
 	timestamp: Joi.number().required(),
 };
 
+const TRANSACTION_EXECUTION_STATUSES = [
+	'pending',
+	'succeeded',
+	'failed',
+];
+
 const transactionSchema = {
```

Comment:
May be we can add pattern check wherever possible and move all `regex` to `regex.js` file.
```suggestion
const block = {
	id: Joi.string().required(),
	height: Joi.number().required(),
	timestamp: Joi.number().required(),
};
```